src/components/login/LoginPage.js

```diff
@@ -1,85 +1,139 @@
 
-import React from 'react';
-import { Link } from 'react-router-dom';
+import React, { Component } from 'react';
+import { Link, Redirect } from 'react-router-dom';
+import { connect } from 'react-redux';
+import GoogleLogin from 'react-google-login';
+import PropTypes from 'prop-types';
+
 import emailIcon from '../../../public/assets/icons/mail-icon.svg';
+import login, { onRequestClick, onFailureClick } from '../../actions/googleLoginActions';
+import loginGoogle from '../../utils/loginGoogle';
 
-const LoginPage = () => (
-  <div>
-    <div className="site-content vertical-center">
-      <div className="container">
-        <div className="row login-wrapper">
-
-          <div className="col-md-4 signup text-center order-md-2">
-            <div className="row welcome-banner align-items-center">
-              <div className="col-md-12">
-                <h2 className="welcome-text">Welcome back!</h2>
-                <p className="text-center welcome-body">To keep sharing your exciting Ideas please log in into your account</p>
-              </div>
-            </div>
+class LoginPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+    };
```

Comment:
why is this on two lines? is the declaration of state even necessary?

Reply:
removed